armada/handlers/armada.py

```diff
@@ -66,8 +66,8 @@ def __init__(self,
         self.wait = wait
         self.timeout = timeout
         self.tiller = Tiller(tiller_host=tiller_host, tiller_port=tiller_port)
-        self.documents = yaml.safe_load_all(file)
-        self.config = self.get_armada_manifest()
+        self.documents = list(yaml.safe_load_all(file))
```

Comment:
@gardlt got it. current problem is the rest of the code isn't expecting it to be in a list. can we say something like `self.documents = list(yaml.safe_load_all(file))[0]` to get the values from the generator but store it as a dict as expected?